Neos.ContentRepository.Core/Classes/Feature/Common/TetheredNodeInternals.php

```diff
@@ -133,12 +133,17 @@ protected function createEventsForMissingTetheredNode(
                 );
             }
 
-            $childNodeSource = null;
-            foreach ($childNodeAggregate->getNodes() as $node) {
-                $childNodeSource = $node;
-                break;
+            $rootDimensionSpacePoints = $this->interDimensionalVariationGraph->getRootGeneralizations();
+            if (empty($rootDimensionSpacePoints)) {
+                // safeguard
+                throw new \RuntimeException(
+                    'The dimension space is empty, please check your configuration.',
+                    1706864056
+                );
             }
-            /** @var Node $childNodeSource Node aggregates are never empty */
+            $arbitraryRootDimensionSpacePoint = array_shift($rootDimensionSpacePoints);
+            $childNodeSource = $childNodeAggregate->getNodeByCoveredDimensionSpacePoint($arbitraryRootDimensionSpacePoint);
```

Comment:
i dont know if this is an improvement, or we introduce `NodeAggregate::getSingleNodeIndependentOfItsDimension` instead.